components/style/properties/shorthand/mask.mako.rs

```diff
@@ -120,6 +120,30 @@
          })
     }
 
+    fn mask_position_to_css<W>(x: &HorizontalPosition,
+                               y: &VerticalPosition,
+                               dest: &mut W)
+                               -> fmt::Result where W: fmt::Write {
```

Comment:
This function looks identical to the one for background position. Could you move the function to `values::specified::position` and share it between the two places?